client/components/info-popover/README.md

```diff
@@ -57,24 +57,29 @@ that can trigger the opening and closing of the InfoPopover then you need to pas
 ```
 
 ```js
-handleAction( event ) {
-	this.refs && this.refs.infoPop._onClick( event );
-},
+const refs = {};
```

Comment:
I think generally we should be using `React.createRef()` more in those examples.